R/MeasureFairness.R

```diff
@@ -0,0 +1,54 @@
+#' @title Fairness Measure
+#' @name mlr_measures_fairness
+#'
+#' @description
+#' This measure specializes [Measure] for Fairness Measure problems:
+#' Users could use [groupwise_abs_diff()], [groupwise_diff()], or [groupwise_quotient()] to evaluate
+#' the fairness measures based on base_measures. For example, the false positive rate bias or
```

Comment:
You are using base_measures as a word but it is not clear what this means.
Trying to improve the description:

```
This measure specializes [Measure] to ...
A common approach to quantifying a model's fairness is to compute the difference between a protected and an unprotected group according to some performance metric such as `classification error`, `false positive rate` or others. 
This measure allows for the composition of a fairness-metric from such a `base_measure` by computing group-wise difference according to some strategy. 
Several options wrt. computing difference such as ... are available.

Reply:
I understand that this is really hard, but if you try to put yourself in the shoes of somebody who has not written the code, it needs to be understandable.

Reply:
A good sentence to also include here is that this implements `statistical group fairness measures`

Reply:
That is really helpful, I have updated the documentation

Reply:
Mark as resolved